### PR TITLE
DISTPG-392 Release notes update 11.15

### PR DESCRIPTION
**Pull request by nastena1606**
new file:   docs/release-notes-v11.15.upd.md
	modified:   docs/release-notes.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/release-notes-v11.15.upd.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  </tbody>
+</table>
+
+Percona Distribution for PostgreSQL is a collection of tools to assist you in managing PostgreSQL. Percona Distribution for PostgreSQL
+installs PostgreSQL and complements it by a selection of extensions that
+enable solving essential practical tasks efficiently.
+
+This update of Percona Distribution for PostgreSQL includes [pg_stat_monitor 1.0.0-rc.2](https://percona.github.io/pg_stat_monitor/REL1_0_STABLE/RELEASE_NOTES.html#100-rc2) - the new version of the statistics collection tool for PostgreSQL.
+
+We welcome your feedback on your experience with `pg_stat_monitor` in the [public JIRA project](https://jira.percona.com/projects/DISTPG).
```
Comment (LenzGr) [inline docs/release-notes-v11.15.upd.md]:
```suggestion
We welcome your feedback on your experience with `pg_stat_monitor` on our [Forum](https://forums.percona.com/c/postgresql/pg-stat-monitor/69) and in the [public JIRA project](https://jira.percona.com/projects/DISTPG).
```

Comment (LenzGr) [reply docs/release-notes-v11.15.upd.md]: Could you please add this to the other versions as well?

Comment (nastena1606) [reply docs/release-notes-v11.15.upd.md]: Done

**Review by ImTheKai — COMMENTED** [review]
Why is the document called docs/release-notes-v12.9.upd.md but the release is 12.8?

**Review by ImTheKai — APPROVED** [review]
lgtm now

**Review by LenzGr — APPROVED** [review]
LGTM, thank you!

**Review by hqakhtar — APPROVED** [review]
LGTM

**Pull request merged**